src/components/ScoreSetHeatmap.vue

```diff
@@ -100,8 +101,10 @@ export default {
     wtVariants: function() {
       return this.wtAminoAcids ? this.prepareWtVariants(this.wtAminoAcids) : []
     },
-    showHeatmap: function() {
-      return this.simpleVariants && this.simpleVariants.length
+    heatmapVisible: function() {
+      const heatmapVisible = this.simpleVariants && this.simpleVariants.length
+      this.$emit('heatmapVisible', heatmapVisible)
```

Comment:
It's probably wise avoid emitting an event from the function that recalculates a computed property. It's a little more code, but I would suggest adding a watcher to emit the event (and maybe the watcher should check whether the value has really changed first).

Aside from the general notion that computed property functions shouldn't have side-effects, it's possible for this function to be called repeatedly with no change in the computed value.